portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,135 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="intro-wrapper">
+      <div id="intro">
+        Hello world, I am
+        <span id="name">Bao Xian</span>
+      </div>
+    </div>
+    
+    <div id="content" class='hide'>
+
+      <div id='main' class='section'>
```

Comment:
it is better to stick with either ' or " for the whole project if the language allows both ' and " to be used in string
(language like java, C, only allows ' for character)

Reply:
Ok! Have changed all to single quote